.gitignore

```diff
@@ -0,0 +1,4 @@
+# Build outputs for sample Meet Add-ons
+addons-web-sdk/samples/dist/
```

Comment:
Could probably simplify these to:
*/dist
*/node_modules
*/.DS_STORE

Reply:
needs two asterisks to include the subdirs, but yeah good idea